orange/src/main/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactory.java

```diff
@@ -83,9 +79,13 @@ private static List<CuppaPrediction> extractSortedProbabilities(@NotNull CuppaPr
                     .altSjCohortClassifier(probabilitiesByClassifier.get(ClassifierName.ALT_SJ))
                     .build();
 
-            cuppaPredictionsOrangeFormat.add(prediction);
+            // If a classifier has no data for a specific cancer type we should remove it completely, see ACTIN-1057
```

Comment:
Feels a bit unusual to refer to a ticket in code, unless its an open TODO that needs followup.

Reply:
indeed I'd normally just include this sort of thing in the commit message. Future devs can annotate the code to see it.

Reply:
Updated

Reply:
Do note that I added a comment because the implementation choice may not be optimal. We could for instance also expose a wider CUPPA datamodel in ORANGE and leave to downstream how to handle NaN likelihoods, but considered that unnecessary at this point